test/sdk/post-processing-streaming-pipeline/multicall-usecase-pipeline-errors.test.ts

```diff
@@ -28,10 +28,33 @@ describe("Post Processing Streaming Pipeline Error Handling", () => {
             throw new Error("Should not be called.");
         }
         
-        transformResponseModel(responseModel: TResponseModel, dto: TDTO): TResponseModel {
+        transformResponseModel(responseModel: TResponseModel, dto: TDTO): TResponseModel | BaseErrorResponseModel {
+            if (dto.status === 'error') return dto as BaseErrorResponseModel;
+            return responseModel;
+        }
+    }
+
+    class OptionalResponsePipelineElement extends BaseStreamingPostProcessingPipelineElement<
```

Comment:
The test had to be changed to feature different types of pipeline elements, as now they themselves determine error handling